pep2html.py

```diff
@@ -569,7 +569,7 @@ def fix_rst_pep(inpath, input_lines, outfile):
 
 def get_pep_type(input_lines):
     """
-    Return the Content-Type of the input.  "text/plain" is the default.
+    Return the Content-Type of the input.  "text/x-rst" is the default.
     Return ``None`` if the input is not a PEP.
```

Comment:
```suggestion
    Ensure the Content-Type is "text/x-rst".
    Raise ValueError if another value is specified.
```

Reply:
I won't oppose you doing so in another PR, but I don't think there's much point in further modifying the ancient `pep2html.py` script, which will hopefully very soon go the way of the dodo, to explicitly refuse to process `text/plain` PEPs, since our linters already check for and fail on them, while all the other, much heftier code is still left in there to process them.